### DISCUSSION TITLE
Lazy connection

### DISCUSSION THREAD

**Discussion by martinos** [jeremyevans/sequel#1964] [General]
I have a have an application that fails to boot when there's a Sequel that can't connect, even if I don't need this database for this execution. In my case I have some thor tasks that does not depends on the Sequel connection but some other does.  

Is there a way to connect to the database  only when a query is made?

**Comment by jeremyevans** [comment]
You can set the `test: false` Database option and Sequel will not attempt to connect when creating the Database object.

**Reply by martinos** [reply]
Awesome, thanks for your quick response.
